src/pages/HomePage/Home.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+function HeroSection() {
+  return (
+    <>
+      <Wrapper>
+        <video src="../../Assets/albania.mp4" autoPlay loop muted />
+        <Header>BALKAN AWAITS</Header>
+        <Text>What are you waiting for?</Text>
+      </Wrapper>
+    </>
+  );
+}
+
+export default HeroSection;
```

Comment:
I think most of the time you write the export default directly, in this case it would be line 4
Also it is confusing that you file name is `Home.js` and the component is called `HeroSection` (which also doesn't make sense to me)